src/wp-includes/class-wp-theme-json.php

```diff
@@ -2750,6 +2769,19 @@ protected static function remove_insecure_styles( $input ) {
 				}
 			}
 		}
+
+		// Ensure indirect properties not handled by `compute_style_properties` are allowed.
+		foreach ( static::INDIRECT_PROPERTIES_METADATA as $property => $path ) {
+			$value = _wp_array_get( $input, $path, array() );
+			if (
+				isset( $value ) &&
```

Comment:
```suggestion
				null !== $value &&
```
This might be a little clearer when reading the conditions:
- If it's not `null`
- and not `array`
- and is a safe CSS declaration